Chaincase.iOS/AppDelegate.cs

```diff
@@ -32,7 +41,7 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
             ZXing.Net.Mobile.Forms.iOS.Platform.Init();
             var formsApp = new BlazorApp(fileProvider: null, ConfigureDi);
             var serviceProvider = formsApp.ServiceProvider;
-
+            ServiceProvider = serviceProvider;
```

Comment:
I think this is a necessary evil because serviceProvider is constructed here and so the services can't be injected. Perhaps we should set services to readonly field rather than hold onto the provider @Kukks ?